packages/internal-scripts/src/release-image.ts

```diff
@@ -166,8 +161,41 @@ EXAMPLES
   });
   console.log(`Wrote screenshot to '${imageFileName}'`);
   await browser.close();
-  process.exit();
-};
+}
+
+function exitWithUsageError() {
+  console.error(
+    `
+USAGE
+  release-image CHANGELOG_PATH
+  release-image (-f CHANGELOG_PATH [-v VERSION])...
+  release-image --releaseContent RELEASE_CONTENTS
+
+EXAMPLES
+  To generate the release image for the reactive-element package:
+
+      release-image packages/reactive-element/CHANGELOG.md
+
+  For multiple packages in a single image:
+
+      release-image reactive-element/CHANGELOG.md lit-html/CHANGELOG.md
+
+  To generate an image composed of specific version numbers, including
+  multiple versions of the same package:
+
+      release-image -f reactive-element/CHANGELOG.md -v 3.2.0 \\
+                    -f lit-html/CHANGELOG.md -v 2.0.1 \\
+                    -f lit-html/CHANGELOG.md -v 2.0.0
+
+  To pass arbitrary contents into the image <body>, use the --releaseContent
+  option (or -r):
+
+      release-image -r "<p>Arbitrary Release Contents</p>"
```

Comment:
Perhaps worth noting the `releaseContent` will be passed through a markdown parser as it is embedded to the body so can be markdown string?

Reply:
^ Great comment!
Additional async suggestion from @bicknellr - Change this flag to accept a markdown file which is inserted instead of raw content.

Reply:
Addressed by changing this to `markdownFile`, which also communicates that markdown is parsed.